client/app/widgets/Resources/index.jsx

```diff
@@ -110,6 +110,13 @@ export class Resources extends React.Component<Props, State> {
     });
   };
 
+  updateTagFilter = (tagLabel: String) => {
+    this.setState((prevState: State) => {
+      const updatedBoxes = prevState.checkboxes.map((checkbox) => checkbox.label === tagLabel ? {...checkbox, checked: true } : checkbox)
+      return { checkboxes: updatedBoxes };
```

Comment:
We should write an integration test for this! You'll want to test that clicking tags changes the checkboxes.